cmd/api/src/model/audit.go

```diff
@@ -156,6 +157,27 @@ type AuditEntry struct {
 	ErrorMsg string
 }
 
+// Necessary function for testing. Ensures all fields except CommitID match so mocking checks pass.
+func (s AuditEntry) Matches(x interface{}) bool {
+	if expected, ok := x.(AuditEntry); !ok {
+		return false
+	} else if s.Action != expected.Action {
+		return false
+	} else if s.ErrorMsg != expected.ErrorMsg {
+		return false
+	} else if s.Status != expected.Status {
+		return false
+	} else if !reflect.DeepEqual(s.Model, expected.Model) {
+		return false
+	}
+
+	return true
+}
+
+func (s AuditEntry) String() string {
```

Comment:
Is this Stringer actually getting any use? I couldn't find any issues by deleting it, so I'm not sure why we need it in this form right now

Reply:
Same as [this comment](https://github.com/SpecterOps/BloodHound/pull/440#discussion_r1504981753).